images/kubectl-build-deploy-dind/scripts/exec-routes-generation.sh

```diff
@@ -0,0 +1,360 @@
+#!/bin/bash
+
+YQ=yq
```

Comment:
I'm going to merge https://github.com/uselagoon/lagoon/pull/2919 first - once that PR is merged:

`yq` = yq v4
`yq3` = yq v3

So you can either change this line to be yq3, or change the YQ commands to be v4 compatible :shrug:

Reply:
using yq3